client/state/products-list/selectors.js

```diff
@@ -1,3 +1,7 @@
+import { property } from 'lodash';
+
+export const getProductsList = property( 'productsList.items' );
```

Comment:
Yes, your example is a perfectly reasonable test. 

> Put in other words, what happens if we were to change the structure of the state tree? 

Our reducer tests would catch that. 😄 

We could go on about using fixtures or factories, but I really don't think it's a big deal to copy test data. It makes reading tests much easier. 

I guess where do you draw the line @dmsnell ? Do you wish we had no tests for any of our redux functions? I personally find and fix bugs when writing tests. Refactoring becomes much simpler too, for example in #7702 I was able to swap in another json validator rather easily since we had good persistence test coverage.